mlrun/serving/states.py

```diff
@@ -1298,9 +1400,16 @@ def add_model(
             model_class.to_dict() if isinstance(model_class, Model) else {}
         )
         if outputs is None and isinstance(
-            model_artifact, mlrun.artifacts.ModelArtifact
+            model_artifact,
+            ModelArtifact,
         ):
             outputs = [feature.name for feature in model_artifact.spec.outputs]
+        elif outputs is None and isinstance(
+            model_artifact,
+            LLMPromptArtifact,
+        ):
+            _model_artifact = model_artifact.model_artifact
+            outputs = [feature.name for feature in _model_artifact.spec.outputs]
```

Comment:
This outputs parsing in only done if the user passed an artifact. What if a uri is used? Will the outputs not be calculated? What are the implications?